src/auth/tenant.ts

```diff
@@ -89,26 +89,21 @@ export class Tenant {
    */
   public readonly displayName?: string;
 
-  /**
-   * The email sign in provider configuration.
-   */
-  public readonly emailSignInConfig?: EmailSignInConfig;
-
-  /**
-   * The multi-factor auth configuration on the current tenant.
-   */
-  public readonly multiFactorConfig?: MultiFactorAuthConfig;
-
   /**
    * The map containing the test phone number / code pairs for the tenant.
    */
   public readonly testPhoneNumbers?: {[phoneNumber: string]: string};
 
+  private readonly _emailSignInConfig?: EmailSignInConfig;
```

Comment:
What's the issue with `public readonly`? Are these not compatible with the type definition generator?

Reply:
Issue is not with `public readonly`. It's the types of these fields. Both `EmailSignInConfig` and `MultiFactorAuthConfig` are internal types, and therefore cannot be exposed in the public API. Their corresponding public interfaces are `EmailSignInProviderConfig` and `MultiFactorConfig`. I've declared the getters with the public types.

Reply:
Ah ok, makes sense. Thanks for explaining.